droopler.install

```diff
@@ -608,6 +608,34 @@ function droopler_update_8103() {
     }
   }
 
+  //Install SEO contrib modules
```

Comment:
We should move this code to a new `hook_update` (to be able to update between beta2 and beta3)